Packs/WhisperGateCVE-2021-32648/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,35 @@
+#### Playbooks
+
+##### New: WhisperGate and HermeticWiper & CVE-2021-32648
```

Comment:
There are duplicates of the changes described in the release note.
Delete the first description ##### New: WhisperGate and HermeticWiper & CVE-2021-32648